examples/mobile/HomeApp/index.html

```diff
@@ -34,8 +34,10 @@
 		</header>
 		<div class="ui-content">
 			<div class="ui-content-area">
-				Weather Widget
-			</div>
+				<a class="weatherwidget-io" href="https://forecast7.com/en/37d57126d98/seoul/" data-label_1="Seoul" data-label_2="South Korea" data-theme="original" >Seoul South Korea</a>
```

Comment:
@lslachciak 
please add style "border-radius: 26px".
so, we will get the results you expect.

Reply:
done